tsdb/engine/tsm1/wal.go

```diff
@@ -129,6 +131,13 @@ func (l *WAL) enableTraceLogging(enabled bool) {
 	}
 }
 
+// setSyncDelay sets the duration to wait before fsync writes.  A value of 0
+// will cause every write to be fsync'd.  This must be called before the WAL
+// is opened.
+func (l *WAL) setSyncDelay(d time.Duration) {
```

Comment:
I'd say just set the field directly, then. Creating an unexported "setter" method just feels wrong since it's not doing anything other than a field assignment.